packages/eslint-plugin-circuit-ui/component-lifecycle-imports/index.ts

```diff
@@ -15,6 +15,8 @@
 
 import { ESLintUtils, TSESTree } from '@typescript-eslint/utils';
 
+/* eslint-disable */
```

Comment:
ESLint doesn't resolve the types for `@typescript-eslint/*` packages. I couldn't figure out why, though I suspect it's related to a dependency cycle since this code is used to lint itself. 

Unfortunately, disabling ESLint is the only workaround I could find.